ambry-cloud/src/main/java/com/github/ambry/cloud/VcrServer.java

```diff
@@ -233,6 +277,12 @@ public void shutdown() {
       if (networkServer != null) {
         networkServer.shutdown();
       }
+      if (requestHandlerPoolForHttp2 != null) {
```

Comment:
Similarly this part is shared too. Can we separate this out into a util method -- `shutDownAndHttp2`